common/src/main/java/com/tencent/rss/common/util/RssUtils.java

```diff
@@ -202,4 +203,34 @@ public static String generateShuffleKey(String appId, int shuffleId) {
   public static String generatePartitionKey(String appId, Integer shuffleId, Integer partition) {
     return String.join(Constants.KEY_SPLIT_CHAR, appId, String.valueOf(shuffleId), String.valueOf(partition));
   }
+
+  public static <T> List<T> loadExtensions(
```

Comment:
The method should target to initialize only one instance

Reply:
use list is more general for it may load multiple extentions.